tests/framework/OutStreams/projectile_mod.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
How different is this model from the one in the analytic test?

Reply:
The main different is the way to determine the number of time steps for each samples. This model will generate different time steps for different samples, which will output an unsync HistorySet.

Reply:
Can we add an argument to pass to "projectile" by way of constant to trigger the choice of time steps instead?

Reply:
agree

Reply:
any news on this?

Reply:
fixed.